sentences/de/_common.yaml

```diff
@@ -486,6 +486,9 @@ lists:
   shopping_list_item:
     wildcard: true
 
+  todo_list_item:
+    wildcard: true
+
```

Comment:
this addition of `todo_list_item` seems to be unrelated to the shopping list topic?